### ISSUE TITLE
Command line going crazy when LanmanServer isn't running

### ISSUE THREAD

**Issue by zinalili** [DNSCrypt/dnscrypt-proxy#340]
If you stop LanmanServer and try to run "service-install.bat" command line windows will open and close in indefinite cycle. I think one error message is enough, no need to tell that eternally :)

**Comment by jedisct1** [comment]
Duplicate, and already fixed :)

**Comment by ghost** [comment]
EDIT: I should have posted hereafter on #336 of which this thread is the duplicate; I read too quickly.

@LaxrMediumWas I had faced the same issue when I first installed dnscrypt-proxy 2.0
I changed the following three bat files and no more issues, here of course on Windows:

**service-install.bat**
```
@echo off
cmd.exe /c "dnscrypt-proxy.exe -service install"
cmd.exe /c "dnscrypt-proxy.exe -service start"
```

**service-restart.bat**
```
@echo off
cmd.exe /c "dnscrypt-proxy.exe -service restart"
```

**service-uninstall.bat**
```
@echo off
cmd.exe /c "dnscrypt-proxy.exe -service stop"
cmd.exe /c "dnscrypt-proxy.exe -service uninstall"
```

Of course you'll have to change Windows DNS settings accordingly (127.0.0.1 when started, your dns when closed).

**Comment by AL3x3y0** [comment]
It's not fixed in 2.0.10.
